apps/dashboard/src/app/event/components/feedback-form-edit-form.tsx

```diff
@@ -0,0 +1,271 @@
+import {
+  type FeedbackFormId,
+  type FeedbackFormWrite,
+  FeedbackFormWriteSchema,
+  FeedbackQuestionSchema,
+  type FeedbackQuestionWrite,
+  FeedbackQuestionWriteSchema,
+} from "@dotkomonline/types"
+import { DragDropContext, Draggable, type DropResult, Droppable } from "@hello-pangea/dnd"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { Icon } from "@iconify/react/dist/iconify.js"
```

Comment:
I *think* we can just use the base import here

```suggestion
import { Icon } from "@iconify/react"
```